GitUI/GitUIExtensions.cs

```diff
@@ -271,5 +272,27 @@ public static Control FindFocusedControl(this ContainerControl container)
                 }
             }
         }
+
+        public static string GetGitVersionString()
+        {
+            if (!File.Exists(AppSettings.GitCommandValue))
+            {
+                return "Not installed";
+            }
+
+            FileVersionInfo versionInfo = FileVersionInfo.GetVersionInfo(AppSettings.GitCommandValue);
+
+            return $"{versionInfo.FileMajorPart}.{versionInfo.FileMinorPart}.{versionInfo.FileBuildPart}.{versionInfo.FilePrivatePart}";
+        }
+
+        public static string GetIssueInfo()
+        {
+            StringBuilder sb = new StringBuilder();
+            sb.AppendFormat("{1}- Git Extensions version:  {0}{1}", AppSettings.ProductVersion, Environment.NewLine);
+            sb.AppendFormat("- GIT version:  {0}{1}", GetGitVersionString(), Environment.NewLine);
+            sb.AppendFormat("- OS version:  {0}{1}", Environment.OSVersion.ToString(), Environment.NewLine);
+            sb.AppendFormat("- .NET version:  {0}{1}", Environment.Version, Environment.NewLine);
```

Comment:
This property's return is pretty meaningless. 
Have a look at [this SO discussion](https://stackoverflow.com/q/49750769/2338036), I think we should be using `RuntimeInformation.FrameworkDescription`  and `AppDomain.CurrentDomain.SetupInformation.TargetFrameworkName`.

Reply:
![image](https://user-images.githubusercontent.com/3629489/47661049-ab340a00-db6e-11e8-8360-e5a26a1813a3.png)
I think RuntimeInformation gives the better description.